libraries/AP_HAL_SITL/AP_HAL_SITL_Namespace.h

```diff
@@ -14,4 +14,14 @@ class Util;
 class Semaphore;
 class GPIO;
 class DigitalSource;
+
+// Number of input channels
+static const uint8_t SITL_INPUT_MAX_CHANNELS = 12;
+
+// Number of rc input channels
+static const uint8_t SITL_RC_INPUT_CHANNELS = 16;
+
+// Number of rc output channels
+static const uint8_t SITL_RC_OUTPUT_CHANNELS = 16;
```

Comment:
I agree with the change from define. But these should rather be on the header they were before, in which they are used (or at least where they logically belong).